human_ratings/README.md

```diff
@@ -0,0 +1,18 @@
+# Human Ratings of Simplifications
+
+In this folder you will find the ratings that where used in section 6 "Evaluating Evaluation Metrics" of the paper.
+
+
+
+## Collection of Human Ratings
+Description extracted from the paper:
+> We randomly chose 100 original sentences from ASSET and, for each of them, we sampled one system simplification. The automatic simplifications were selected so that the distribution of simplification transformations (e.g. sentence splitting, compression, paraphrases) would match that from human simplifications in ASSET. That was done so that we could obtain a sample that has variability in the types of rewritings performed. For each sentence pair (original and automatic simplification), we crowdsourced 15 human ratings on fluency (i.e. grammaticality), adequacy (i.e. meaning preservation) and simplicity, using the same worker selection criteria and HIT design of the Qualification Test as in § 5.1.
+
+## File format
```

Comment:
Would it be possible to include the name of the system too?

Reply:
I sadly didn't keep it...
I could find it by brute-forcing but it would be time consuming...

Reply:
Don't worry about it then.